src/routes/EventView.svelte

```diff
@@ -77,6 +77,7 @@
 			</h2>
 		{/if}
 
+		<!-- eslint-disable svelte/no-at-html-tags -->
 		<div class="flex flex-col gap-4 mt-4">{@html description}</div>
```

Comment:
Can we fix this properly? Running [dompurify](https://www.npmjs.com/package/dompurify) should be enough.